### ISSUE TITLE
boto determine account id

### ISSUE THREAD

**Issue by FernandoMiguel** [duo-labs/cloudmapper#53]
could this use 

```
import boto3
print(boto3.client('sts').get_caller_identity()['Account'])
```
to determine the account id it is running ?

**Comment by 0xdabbad00** [comment]
Makes sense. I'm actually not currently using the account id for anything, so I'll look to integrating this concept or removing the account id entirely from the config.

**Comment by jcornell123** [comment]
Could it integrate with Organizations and leverage the ListAccounts API to retrieve all accounts in the Org and auto generate the accounts?  https://docs.aws.amazon.com/organizations/latest/APIReference/API_ListAccounts.html

**Comment by 0xdabbad00** [comment]
@jcornell123 That's a possibility. My current thinking for #286 is to just remove the `config.json` or possibly only look at it in cases where it's data might be used to supersede info acquired from collect.
